### ISSUE TITLE
Lecture notes Header Style 

### ISSUE THREAD

**Issue by aak2054** [kevinaangstadt/CS340-S23-Notes#141]
Is there a way we can come in agreement on having similar header style for our notes, so, all of our lecture notes will have a consistent header style format.

**Comment by CharlesReinhardt** [comment]
I agree, and would highlight or tag this issue if I could! This is something we should look into fixing as our second round of PRs get merged and our main branch lecture notes have their content included.
